vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/rest/HtsgetVcfController.java

```diff
@@ -53,145 +57,170 @@
 
     private Properties evaProperties;
 
-    @Autowired
     private VariantSourceService variantSourceService;
-    @Autowired
+
     private VariantWithSamplesAndAnnotationsService variantService;
 
-    public HtsgetVcfController() throws IOException {
+    public HtsgetVcfController(VariantSourceService variantSourceService,
+                               VariantWithSamplesAndAnnotationsService variantService) throws IOException {
+        this.variantSourceService = variantSourceService;
+        this.variantService = variantService;
         evaProperties = new Properties();
         evaProperties.load(VcfDumperWSServer.class.getResourceAsStream("/eva.properties"));
     }
 
     @RequestMapping(value = "/{id}", method = RequestMethod.GET, consumes = "application/*",
             produces = "application/vnd.ga4gh.htsget.v0.2rc+json; charset=UTF-8")
     public ResponseEntity getHtsgetUrls(
+            @ApiParam(value = "Identifiers of studies, e.g. PRJEB9799. Each individual identifier of " +
+                    "studies can be looked up in https://www.ebi.ac.uk/eva/webservices/rest/v2/studies?species=" +
+                    "ecaballus&assembly=20&pageNumber=0&pageSize=20 in the field named 'studyId'.", required = true)
             @PathVariable("id") String id,
+            @ApiParam(value = "Format in which the data will be represented, e.g. VCF", defaultValue = "VCF")
             @RequestParam(name = "format", required = false) String format,
+            @ApiParam(value = "Reference sequence name, e.g. 1")
```

Comment:
Since the variant warehouse has a mix of representation for this, should we say "1 or chr1 or CM000001.1" - something like that?